Makefile

```diff
@@ -452,11 +452,8 @@ site/src/api/typesGenerated.ts: scripts/apitypings/main.go $(shell find codersdk
 docs/admin/prometheus.md: scripts/metricsdocgen/main.go scripts/metricsdocgen/metrics
 	go run scripts/metricsdocgen/main.go
 	cd site
-ifdef CI
-	yarn run format:check
-else
 	yarn run format:write
-endif
+.PRECIOUS: docs/admin/prometheus.md
```

Comment:
We don't use this for other tasks so I don't think we should use it here otherwise devs may commit unformatted docs. I think removing the format:check should be good enough.

Can you change the format:write to only do it on the single file as well?

Reply:
In theory, when `make` receives a signal to quit/kill, it should delete all target files, so in this case `PRECIOUS` seems to be reasonable.

I can try without the marker and see if it's sufficient. I can iterate on it if it isn't.